src/advanced-concepts/custom_signature_validation.md

```diff
@@ -0,0 +1,17 @@
+# Custom Signature Validation Scheme
+
+Account Abstraction on Starknet supports various signature schemes. This means that signature schemes on Starknet are not limited to just one, any standard signature scheme can be validated, for example Starknet signature, Secp256k1, Secp256r1, Eip191 et al are some of the custom signatures that can be validated on Starknet currently. 
+
+### The Concepts of Accounts and Signers
+
+i. **Account:** All accounts are smart contracts that can hold assets and execute transactions on Starknet protocol, these account contracts however must implement some specific methods outlined in SNIP-6.
```

Comment:
Add a link to [Account Contract](https://starknet-by-example.voyager.online/advanced-concepts/account_abstraction/account_contract.html)